test/csuite/random_directio/smoke.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
+set -e
+
+# Smoke-test random-abort as part of running "make check".
+
+echo TEST_WRAPPER = $TEST_WRAPPER
+$TEST_WRAPPER ./test_random_directio  -t 5 -T 5
```

Comment:
Given all the differently options the program supports, I would like to see `smoke.sh` expanded greatly to run as many of them as is practical. Even if they're turning up bugs, having them in there, but commented out (like `timestamp_abort/smoke.sh` comments out the LSM variants because LSM does not work with timestamps for now).